snappy/converge.go

```diff
@@ -51,8 +53,27 @@ type response struct {
 	Message string `json:"message"`
 }
 
+func (h *Handler) getSnap(name string) (*client.Snap, error) {
+	snap, _, err := h.snapdClient.Snap(name)
+	if err == nil {
+		return snap, nil
+	}
+
+	// Snap() now only returns installed snaps so search the store as a fallback
+	snaps, _, err := h.snapdClient.FindSnaps(name)
+	if err != nil {
+		return nil, err
+	}
+
+	if !(len(snaps) > 0 && strings.EqualFold(snaps[0].Name, name)) {
```

Comment:
I cannot confirm that; the documentations don't make any such promise afaik.
If you need exact matches you can ask for them though